src/ducks/settings/TogglePane.jsx

```diff
@@ -1,83 +1,31 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
+import React from 'react'
```

Comment:
You can import PureComponent to not repeat React ?
```suggestion
 import React, { PureComponent } from 'react'
```